tests/baselines/reference/self-types-case-insensitive.errors.txt

```diff
@@ -0,0 +1,50 @@
+tests/cases/compiler/self-types-case-insensitive.ts(18,11): error TS18051: Type '"Acept"' is not assignable to type 'CaseInsensitive<"Set-Cookie" | "Accept">'
+  Type 'Lowercase<"Acept">' is not assignable to 'Lowercase<"Set-Cookie" | "Accept">'
+    Type '"acept"' is not assignable to '"set-cookie" | "accept"'
+tests/cases/compiler/self-types-case-insensitive.ts(32,3): error TS2322: Type '{ "Set-Cookie": string; }' is not assignable to type 'Headers'.
+  Object literal may only specify known properties, and '"Set-Cookie"' does not exist in type 'Headers'.
+
+
+==== tests/cases/compiler/self-types-case-insensitive.ts (2 errors) ====
+    type CaseInsensitive<T extends string> =
+      self extends string
+        ? Lowercase<self> extends Lowercase<T>
+            ? self
+            : Never<[
+              `Type '${Print<self>}' is not assignable to type 'CaseInsensitive<${Print<T>}>'`,
+              `Type 'Lowercase<${Print<self>}>' is not assignable to 'Lowercase<${Print<T>}>'`,
+              `Type '${Print<Lowercase<self>>}' is not assignable to '${Print<Lowercase<T>>}'`

```

Comment:
Might prevent from different error message styles like "Can not assign {} to {}", "Must be {}, got {}" etc.

```suggestion
              NotAssignableErorrMessage<`${Print<self>}`, `CaseInsensitive<${Print<T>}>`>,
              NotAssignableErorrMessage<`Lowercase<${Print<self>}>`, `Lowercase<${Print<T>}>`,
              NotAssignableErorrMessage<`${Print<Lowercase<self>>}`, `${Print<Lowercase<T>>}`>
```